### PR TITLE
Feature/67 render mural marker

### PR DESCRIPTION
**Pull request by SiboYang**
# Summary
implemented render the mural markers to the map, added cors in backend to run backend and frontend at the same time locally (not sure if this is the way to do that....)
image

## Test Plan
manually

## Related Issues

#67

### CODE REVIEW

File: backend/src/app.ts [hunk 1]
```diff
@@ -16,6 +16,8 @@ class App {
     this.associationSetup.sync();
   }
   private config(): void {
+    const cors = require('cors')
+    this.app.use(cors())
```
Comment (tedspare) [inline backend/src/app.ts:20]: thanks for doing this!

File: frontend/src/App/App.tsx [hunk 2]
```diff
@@ -12,20 +12,33 @@ function App() {
...
+  const getMural = async () => {
+    const response = await fetch("http://localhost:3000/mural");
+    const data = await response.json();
+
+    setMurals(data.murals.rows);
+  };
+
+  useEffect(() => {
+    getMural();
+  }, []);
```
Comment (tedspare) [inline frontend/src/App/App.tsx:34]: nice solution

File: frontend/src/Map/Map.tsx [hunk 3]
```diff
@@ -1,43 +1,56 @@
...
-mapboxgl.accessToken = process.env.REACT_APP_MAPBOX_TOKEN || '';
+mapboxgl.accessToken = process.env.REACT_APP_MAPBOX_TOKEN || "";
 
 interface IMapProps {
-    mapContainer: HTMLElement | string | null;
+  mapContainer: HTMLElement | string | null;
+  murals: any;
 }
 
 function Map(props: IMapProps) {
```
Comment (tedspare) [inline frontend/src/Map/Map.tsx:17]: we can "spread" the props by writing `function Map({mapContainer, murals} as IMapProps)`

**Review by tedspare — APPROVED** [review]
LGTM

**Comment by tedspare** [comment]
for reference:

image

feel free to merge!

File: backend/src/app.ts (hunk 1 above)
Comment (tedspare) [reply backend/src/app.ts:20]: @JDziewonski98 this is a fairly important backend change - is it good by you?

Comment (JDziewonski98) [reply backend/src/app.ts:20]: will have to change this come deployment but its not a big deal right now

**Pull request merged**